encrypted-media-respec.html

```diff
@@ -134,13 +134,15 @@
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/192">Issue 192</a> - The specification should do more to encourage/ensure CDM-level interoperability.</p>
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/221">Issue 221</a> - This specification contains sections for describing <a href="#security">security</a> and <a href="#privacy">privacy</a> considerations. These sections are not final and review is welcome.</p>
 
-      <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
-      </p>
+      <p>The following features are <strong>at risk</strong> and may be removed:
+	  </p>
```

Comment:
nit: Incorrect indent. Actually, though, we can just move it to the previous line since it's a single sentence.